src/protocol/requests/initProducerId/index.js

```diff
@@ -0,0 +1,12 @@
+const versions = {
+  0: ({ transactionalId, transactionTimeout }) => {
+    const request = require('./v0/request')
+    const response = require('./v0/response')
+    return { request: request({ transactionalId, transactionTimeout }), response }
```

Comment:
What do you think about a default `transactionTimeout` here? Maybe `5000`.

Reply:
I'm in favor of providing a default. I suppose it could go here _or_ in the broker... but I'll add here for now